### ISSUE TITLE
[FEAT]:  Expert Train & Inference for Mustard dataset with ALBEF

### ISSUE THREAD

**Issue by Jasonqi146** [lwaekfjlk/mmodel-soup#90]
### Description

Use Mustard training split of R, U, and S to train model ALBEF seperately and save to

Sarc R finetuned ALBEF
Sarc U finetuned ALBEF
Sarc S finetuned ALBEF
Then record the performance of the three models on the test set.
